iam/snippets/src/main/java/GetRole.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.google.api.gax.rpc.NotFoundException;
+import com.google.cloud.iam.admin.v1.IAMClient;
+import com.google.iam.admin.v1.GetRoleRequest;
+import com.google.iam.admin.v1.Role;
+import java.io.IOException;
+
+// [START iam_get_role]
+public class GetRole {
+
+  public static void main(String[] args) {
+    // TODO(developer): Replace the variable before running the sample.
+    String roleId = "roles/iam.roleViewer";
```

Comment:
Is this a correct example of a role id? Should it be in the format of `projects/PROJ/roles/<ROELNAME>`?

Reply:
ditto.

Reply:
We may want Sita to weigh in on consistency. I think we should be careful about role name and role id. This string may just signal a "role" (https://cloud.google.com/iam/docs/reference/rest/v1/roles/get)